### ISSUE TITLE
Cross reference system registers

### ISSUE THREAD

**Issue by yrp604** [Vector35/binaryninja-api#2210]
The ability to xref system registers would be a useful feature addition.

For example: if I have `msr vbar_el3, x0`, the ability to select `vbar_el3` and hit 'x', then see all other use sites.

**Comment by plafosse** [comment]
Would persistent global search be an acceptable alternative?

**Comment by yrp604** [comment]
Kind of? Like it would provide a UI based way to get the info so it would be an improvement, but conceptually I'd still think of of this as the same type of action as "what write to this memory" and be surprised when the workflow was different.

**Comment by plafosse** [comment]
Yeah the problem here is that we don't have separate system registers and gprs, so we'd have to record all registers which would be expensive. Or generate the list on demand which would be slow, similar to search.
